core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -1740,28 +2480,167 @@ public void testGetAllStoreSomeMatchCacheAccessExceptionBeforeLoaderAllMatchNone
    * </ul>
    */
   @Test
-  public void testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() throws Exception {
-    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B), Collections.singleton("keyA3"));
+  public void testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B), Collections.singleton("keyA3"));
+    this.store = spy(fakeStore);
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_A, KEY_SET_C, KEY_SET_D));
```

Comment:
Do not start in an inconsistent state: the loader needs to know a super set of what the store knows. If that makes the test a duplicate, delete it.